### PR TITLE
Minor fix in doc for [unfold]

### LINKED ISSUE #9634: `eval unfold x in x` does other reduction than unfolding x
#### Description of the problem

```coq
Definition x := 0.
Definition g := let x := x+x in x.
Definition f :=
  let x := x+x in let x := x+x in let x := x+x in
  let x := x+x in let x := x+x in let x := x+x in
  let x := x+x in let x := x+x in let x := x+x in
  let x := x+x in let x := x+x in let x := x+x in
  let x := x+x in let x := x+x in let x := x+x in
  let x := x+x in let x := x+x in let x := x+x in
  let x := x+x in let x := x+x in let x := x+x in
  let x := x+x in let x := x+x in let x := x+x in x.
Goal True.
  let t := eval unfold g in g in idtac t. (* (x + x) *)
  Timeout 1 let t := eval unfold f in f in idtac t. (* Timeout! *)
  exact I.
Qed.
```

#### Coq Version

8.9.0

@JasonGross has this always been the case?

### PR DESCRIPTION
**Pull request by SkySkimmer**
Close #9634

### CODE REVIEW

File: doc/sphinx/proof-engine/tactics.rst [hunk 1]
```diff
@@ -3281,11 +3281,13 @@ the conversion in hypotheses :n:`{+ @ident}`.
...
    :ref:`gallina-definitions` and
    :ref:`vernac-controlling-the-reduction-strategies`). The tactic
-   :tacn:`unfold` applies the :math:`\delta` rule to each occurrence of
-   the constant to which :n:`@qualid` refers in the current goal and
-   then replaces it with its :math:`\beta`:math:`\iota`-normal form.
+   :tacn:`unfold` applies the :math:`\delta` rule to each occurrence
+   of the constant to which :n:`@qualid` refers in the current goal
+   and then replaces it with its :math:`\beta\iota\zeta`-normal form.
+   Use the general reduction tactics if you want to avoid this final
+   reduction, for instance :g:`cbv delta [@qualid]`.
```
Comment (SkySkimmer) [inline doc/sphinx/proof-engine/tactics.rst]: >`` :g:`cbv delta [@qualid]` ``

not sure if this is the correct way to write this in sphinx, @coq/doc-maintainers ?

Comment (cpitclaudel) [reply doc/sphinx/proof-engine/tactics.rst]: I think you want `:n:`

Comment (Zimmi48) [reply doc/sphinx/proof-engine/tactics.rst]: @SkySkimmer ping

**Comment by Zimmi48** [comment]
@SkySkimmer ping

**Comment by SkySkimmer** [comment]
Thanks for the ping, I missed @cpitclaudel's message.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**